modules/gdscript/gdscript_parser.cpp

```diff
@@ -4371,6 +4371,14 @@ bool GDScriptParser::export_annotations(AnnotationNode *p_annotation, Node *p_ta
 		export_type.type_source = variable->datatype.type_source;
 	}
 
+	bool is_dict = false;
```

Comment:
I can't speak for the other types of export annotations, but this is still needed for the basic `@export` scenario; otherwise, all exported dictionaries would show up as untyped in the inspector